scaladoc/src/dotty/tools/scaladoc/SocialLinks.scala

```diff
@@ -19,5 +20,8 @@ object SocialLinks:
       case "gitter" => Left(errorPrefix + "For 'gitter' arg expected one argument: url")
       case "discord" if splitted.size == 2 => Right(Discord(splitted(1)))
       case "discord" => Left(errorPrefix + "For 'discord' arg expected one argument: url")
+      case "custom" if splitted.size == 4 => Right(Custom(splitted(1), splitted(2), splitted(3)))
+      case "custom" if splitted.size == 3 => Right(Custom(splitted(1), splitted(2), splitted(2)))
```

Comment:
Ah, so this answers my question from above. If they don't provide all of them, then you just use the icon for both right?

Reply:
Yes, that's right, dark mode is optional, so if it's not provided, the icon will be used for both modes.

Reply:
Example: 
```social-links:custom::https://www.epfl.com::epfl.png,custom::scala.com::scala_logo.png``` works like
```social-links:custom::https://www.epfl.com::epfl.png::epfl.png,custom::scala.com::scala_logo.png::scala_logo.png```